pkg/rehearse/jobs.go

```diff
@@ -219,9 +230,82 @@ func ConfigureRehearsalJobs(toBeRehearsed config.Presubmits, ciopConfigs config.
 		}
 	}
 
+	// for the remaining cluster types of each template, pick a `random` corresponding job to rehearse.
+	for templateFile, clusterTypes := range templatesWithClusterTypes {
+		for clusterType := range clusterTypes {
+			if job := pickTemplateJob(prConfig.JobConfig.Presubmits, templateFile, clusterType); job != nil {
+
+				// change the configmap name with the temporary created one.
```

Comment:
This seems mostly duped from above?

Reply:
Yea shortly.  I will put the logic in a different function.